src/App.css

```diff
@@ -192,7 +192,7 @@ hr {
 }
 
```

Comment:
- Kindly, use the given font family to stitch the given [Zeppelin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design and have a better look 👌
### Here is your design
![2023-04-06_11-06](https://user-images.githubusercontent.com/52010789/230314429-37c86ec1-9912-44d7-9fe8-96c6ddbd3954.png)
### Zeppelin Design
![2023-04-06_11-02](https://user-images.githubusercontent.com/52010789/230314574-ed5aee23-1ad0-4e50-9b86-ef49880b8f93.png)